src/stores/releases/actions.ts

```diff
@@ -0,0 +1,56 @@
+import GiantSwarm from 'giantswarm';
+import { FlashMessage, messageTTL, messageType } from 'lib/flashMessage';
+import { IState } from 'reducers/types';
+import cmp from 'semver-compare';
+
+import { createAsynchronousAction } from '../asynchronousAction';
+import { IReleaseActionResponse } from './types';
+
+export const loadReleases = createAsynchronousAction<
+  void,
+  IState,
+  IReleaseActionResponse
+>({
+  actionTypePrefix: 'RELEASES',
```

Comment:
I believe it would make more sense to have the base action name something like `RELEASES_LOAD`, so we can leave room for other release actions.

Reply:
Ah yeah, `actionTypes.js` also defines these. I forgot to clean this up